docs/Documentation/Development/Languages/Julia/index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Julia
+has_children: true
+parent: Programming Languages
+---
+
+# Julia
+
+*Julia is a dynamic programming language that offers high performance while being easy to learn and develop code in.*
+
+This section contains demos (in the form of scripts and notebooks) and how to guides for doing various things with Julia on Eagle (and to some extent any HPC environment).
```

Comment:
Is it possible to make this not as system specific? Will these tutorials also work on Swift and Kestrel when ready? 

There is a "Running Julia" section on the "Running on Swift" page which should be transferred out of that page and captured here. https://nrel.github.io/HPC/Documentation/Systems/Swift/running/#running-julia